volunteer.html

```diff
@@ -0,0 +1,56 @@
+</div>
+<div class="container featured-content">
+        <div class="row">
+            <div class="col-12 col-md-7 col-xl-8 hero-media">
+                <h2>Why Volunteer With Us ?</h2>
+                <ul>
+                    <li><b>Support Women in Tech</b> Help create an inclusive environment where women can thrive in technology.</li>
+                    <li><b>Share Your Expertise</b> Use your knowledge and skills to mentor others, lead workshops, or support our events.</li>
+                    <li><b>Build Connections</b> Join a network of like-minded individuals who are passionate about making a positive impact.</li>
+                    <li><b>Develop New Skills</b> Gain valuable experience in teaching, mentoring, event planning, and more.</li>
+                </ul>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-12 col-md-7 col-xl-8 hero-media">
+                <h2>Volunteer Opportunities At A Glance</h2>
+                <ul>
+                    <li><b>Organisational Skills </b> Assist in organizing meetups, online tech events, and other community activities.</li>
+                    <li><b>Show Off Your Tech Skills </b> Collaborate with us to build our new platform or enhance features on our existing website.</li>
+                    <li><b>Enhance Your Writing Skills </b> Contribute to our content creation efforts, while connecting with others who share your passion for positive change.</li>
+                    <li><b>Develop Your Speaking Skills</b> Share your insights with the community through talks or presentations. We’ll host an event for you, and offer support to help you get started if needed.</li>
+                </ul>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-12 col-md-7 col-xl-8">
+                <h2>Kickstart Your Volunteer Journey</h2>
+                <p> Here’s the pocketbook to help you get started on your volunteering journey with us. Dive in and find everything you need to begin making an impact!</p>
+                <div class="card-deck">
+                    {% for resources in site.data.volunteer_resources %}
+                    <div class="card-column col-12 col-lg-8">
+                        <div class="card-resource card card-s card-horizontal">
+                            <div class="card-media col-5">
+                                <img src="{{ resources.image.path }}" alt="{{ resources.image.alt }}">
+                            </div>
+                            <div class="card-body col-7">
+                                <h2 class="card-title">{{resources.title}}</h2>
+                                <a href={{resources.link}} class="ext-link" target="_blank" rel="noopener">Open <span class="icon">{% include icons/arrow-right.svg %}</span></a>
+                            </div>
+                        </div>
+                    </div>
+                    {% endfor %}
```

Comment:
I think is important to include this information here as well: 

![image](https://github.com/user-attachments/assets/91324da8-7f6a-4d83-981a-eab4d4096b99)
it is in about us now: https://womencodingcommunity.com/about

Reply:
Added this now 
![Screenshot 2024-08-29 at 10 42 59](https://github.com/user-attachments/assets/922f5b2f-be03-4d95-a1e8-302acd0eabfe)